libraries/AP_Mission/AP_Mission.h

```diff
@@ -184,6 +184,12 @@ class AP_Mission {
         uint8_t relative_angle; // 0 = absolute angle, 1 = relative angle
     };
 
+    // LOITER_TO_ALT structure
+    struct PACKED Loiter_To_Alt_Command {
```

Comment:
This is to large to fit in storage. How are you handling this? You only get 2 bytes past the location struct, anything else is corrupted/dropped.

Reply:
I'm not handling it. It's broken. I'm a dumbass

Reply:
This issue https://github.com/ArduPilot/ardupilot/issues/2391 is looooong overdue